x/sqlbuilder/select_statement_test.go

```diff
@@ -306,6 +306,26 @@ func TestSelectQuery(t *testing.T) {
 			vs:   map[string]interface{}{"biz": "bar"},
 			args: []interface{}{"bar"},
 		},
+		{
+			name: "exists",
+			ss: SelectStatement{
+				Table:         "foo",
+				SelectClauses: []Marker{Column("biz")},
+				WhereClause: &Exists{
+					Table: "bar",
+					WhereClause: And(
+						EqMarkers(
+							ColumnWithTable("bar_biz", "bar", "biz"),
+							ColumnWithTable("biz", "foo", "biz"),
+						),
+						StaticEq(ColumnWithTable("bar_baz", "bar", "baz"), "qux"),
```

Comment:
```suggestion
						Eq(ColumnWithTable("bar_baz", "bar", "baz")),
```

you are already passing qux L326